sos/report/plugins/ceph_mgr.py

```diff
@@ -36,53 +38,36 @@ class CephMGR(Plugin, RedHatPlugin, UbuntuPlugin):
 
     plugin_name = 'ceph_mgr'
     profiles = ('storage', 'virt', 'container', 'ceph')
-    files = ('/var/lib/ceph/mgr/*', '/var/lib/ceph/*/mgr*')
+    files = ('/var/lib/ceph/mgr/*', '/var/lib/ceph/*/mgr*',
+             '/var/snap/microceph/common/data/mgr/*')
     containers = ('ceph-(.*-)?mgr.*',)
 
     def setup(self):
+        microceph_pkg = self.policy.package_manager.pkg_by_name('microceph')
 
-        self.add_file_tags({
-            '/var/log/ceph/(.*/)?ceph-mgr.*.log': 'ceph_mgr_log',
-        })
-
-        self.add_forbidden_path([
-            "/etc/ceph/*keyring*",
-            "/var/lib/ceph/**/*keyring*",
-            "/var/lib/ceph/**/osd*",
-            "/var/lib/ceph/**/mon*",
-            # Excludes temporary ceph-osd mount location like
-            # /var/lib/ceph/tmp/mnt.XXXX from sos collection.
-            "/var/lib/ceph/**/tmp/*mnt*",
-            "/etc/ceph/*bindpass*",
-        ])
-
-        self.add_copy_spec([
-            "/var/log/ceph/**/ceph-mgr*.log",
-            "/var/lib/ceph/**/mgr*",
-            "/var/lib/ceph/**/bootstrap-mgr/",
-            "/run/ceph/**/ceph-mgr*",
-        ])
-
-        # more commands to be added later
         ceph_mgr_cmds = ([
             "balancer status",
-            "orch host ls",
-            "orch device ls",
-            "orch ls",
-            "orch ls --export",
-            "orch ps",
-            "orch status --detail",
-            "orch upgrade status",
-            "log last cephadm"
+            "log last cephadm",
+            "mgr dump",
+            "mgr metadata",
+            "mgr module ls",
+            "mgr stat",
+            "mgr versions"
         ])
 
-        self.add_cmd_output(
-            [f"ceph {cmd}" for cmd in ceph_mgr_cmds])
-        # get ceph_cmds again as json for easier automation parsing
-        self.add_cmd_output(
-            [f"ceph {cmd} --format json-pretty" for cmd in ceph_mgr_cmds],
-            subdir="json_output",
-        )
+        # if orchestrator is configured
+        orch_configured = self.exec_cmd('ceph orch status')
+        if orch_configured['status'] == 0:
+            orch_mgr_cmds = ([
```

Comment:
You can append to the list here:
```
ceph_mgr_cmds + = ([
...
```

Then you woudn't need
```
ceph_mgr_cmds = ceph_mgr_cmds + orch_mgr_cmds
``

Reply:
ack thank you, done!